src/runtime/config/configuration-acrn.toml.in

```diff
@@ -189,6 +189,21 @@ internetworking_model="@DEFNETWORKMODEL_ACRN@"
 # (default: true)
 disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 
+# disable applying SELinux on the container process
+# If set to false, the type `container_t` is applied to the container process by default.
+# Note: To enable guest SELinux, the guest rootfs must be CentOS that is created and built
+# with `SELINUX=yes`.
+# (default: true)
+disable_guest_selinux=@DEFDISABLEGUESTSELINUX@
+
+# Apply a custom SELinux security policy to the container process inside the VM.
+# This is used when you want to apply a type other than the default `container_t`,
+# so general users should not uncomment and apply it.
+# (format: "user:role:type")
+# Note: You cannot specify MLS policy with the label because the sensitivity levels and
+# categories are determined automatically by high-level container runtimes such as containerd.
+#guest_selinux_label="@DEFGUESTSELINUXLABEL@"
```

Comment:
I am not sure if we want to provide this as a config option at all, I dont see the need for the end-user changing this. We can add this at a later point if a specific need arises.

Reply:
Yes, as I mentioned in #4812, I'm also not sure if the config is helpful for Kata users. 
The users cannot modify SELinux security policy for containers through the [Kubernetes' security context](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#selinuxoptions-v1-core), so this config will be an alternative way for modifying the policy. But I agree with your idea that adds this at alter point when a specific need arises.

> 3. Allow users to apply a custom SELinux security policy to a container
As explained above, the security policy is overridden for VMM processes. Hence, users cannot set a custom policy other than `container_t` through Kubernetes's configuration. To solve the issue, we add `guest_selinux_label` option in the runtime configuration. Users can apply a custom policy using the configuration except for MCS. This feature is helpful for users who want to loosen restrictions, but I don't know actually if any users want to apply custom policies to a container other than `container_t`.